### DISCUSSION TITLE
[WARNING] ... does not support HTTP/2

### DISCUSSION THREAD

**Discussion by MojtabaFaal** [DNSCrypt/dnscrypt-proxy#2095] [Q&A] [answered]
Hi
I receive this **WARNING** error :

[2022-04-17 15:49:08] [WARNING] [controld-uncensored] does not support HTTP/2

[2022-04-17 15:49:40] [WARNING] [controld-unfiltered] does not support HTTP/2

This warning did not exist before
What is the problem ?
Thank

**Comment by jedisct1** [comment]
It means that their servers don't support HTTP/2 :)

Probably a configuration error, or they are using old software. Not a problem per se. except that performance will be poor.

**Reply by MojtabaFaal** [reply]
Can it be removed by you as long as it supports HTTP / 2?

**Reply by MojtabaFaal** [reply]
@jedisct1

**Reply by jedisct1** [reply]
Best thing would be to let them know.

**Comment by lifenjoiner** [comment, marked as answer]
As you can see, there are many servers listed in the `public-resolvers.md`. They will be validated and ranked before being used. This warning is most likely happening during this period. So, it should not be a big deal. Only the best n servers will be adopted.

If you don't like to see any warning, put the servers in `disabled_server_names` of your config file:
```tmol
disabled_server_names = ['controld-uncensored', 'controld-unfiltered']
```
